Neos.ContentRepository.Core/Classes/EventStore/InitiatingEventMetadata.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\EventStore;
+
+use Neos\ContentRepository\Core\SharedModel\User\UserId;
+use Neos\EventStore\Model\Event\EventMetadata;
+
+/**
+ * @internal
+ */
+final readonly class InitiatingEventMetadata
```

Comment:
new utility to encapsulate access instead of passing magic strings around :)

see https://github.com/neos/neos-development-collection/pull/5301/commits/4ee595ebde08bce4686fd185b4d712c5c0555e76